ptp/components/losses/nll_loss.py

```diff
@@ -45,8 +45,15 @@ def __init__(self, name, config):
         # Get number of targets dimensions.
         self.num_targets_dims = self.config["num_targets_dims"]
 
+        # Get the optional ignore_index. -100 is the default value in PyTorch
+        self.ignore_index = -100
+        try:
+            self.ignore_index = self.globals["ignore_index"]
+        except KeyError:
+            pass
+
```

Comment:
I am not sure that I am implementing *optionnal* globals correctly, or even if that is an OK practice to have an optionnal entry at all.
In any case, I couldn't use the `.get()` method because of #26 .

Reply:
interesting... this is the first case when we need global that may not be set before by other component...